src/stylesheets/includes/_layout.scss

```diff
@@ -226,7 +226,8 @@ body {
     &.ui.button,
     &.ui.button:hover,
     &.ui.button:focus {
```

Comment:
you change it for all buttons of the site, if it that you want to do?

Reply:
The changes are ony for:
.donate-button
.vh-button
And they are used only for the Donnation and Virtual Home buttons:
![image](https://user-images.githubusercontent.com/9771/164121397-765efd87-c385-47d9-8bd6-b20059aa1d80.png)